src/Repository/YesplanEventRepository.php

```diff
@@ -52,7 +52,7 @@ public function configureOptions(OptionsResolver $resolver): void
     public function findOldEvents(): array
     {
         $entityManager = $this->getEntityManager();
-        $timeNow = new DateTime('NOW');
+        $timeNow = new \DateTime('NOW');
```

Comment:
Make a habit of using `DateTimeImmutable`. Using `DateTime` (which is mutable) sometimes makes things harder to do (not in this case however).